components/LinksHomepage/index.js

```diff
@@ -52,20 +52,31 @@ export default function LinksHomepage() {
     <>
       <StyledWrapper>
         <StyledForm onSubmit={handleSubmit}>
+ apicategpries
+          <StyledTitle id="title-id" aria-labelledby="title-label">
+            Add Recipe
+          </StyledTitle>
+          <StyledLabel htmlFor="link">Link</StyledLabel>
+
+          <StyledInput
+            id="link"
```

Comment:
I think the `StyledInput`is missing a closing angle:
```suggestion
          <StyledInput
            id="link" >
```